packages/snap/src/index.test.ts

```diff
@@ -0,0 +1,52 @@
+import { installSnap } from '@metamask/snaps-jest';
+import { expect } from '@jest/globals';
+import { panel, text } from '@metamask/snaps-ui';
+
+describe('onRpcRequest', () => {
+  it('hello', async () => {
```

Comment:
```suggestion
  it('supports the "hello" method', async () => {
```
or something similar?

Reply:
Oops, this was intended to be a `describe` block 😅